surveyor/observation_test.go

```diff
@@ -0,0 +1,97 @@
+// Copyright 2019 The NATS Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package surveyor
+
+import (
+	"encoding/json"
+	"testing"
+	"time"
+
+	server "github.com/nats-io/nats-server/v2/server"
+	st "github.com/nats-io/nats-surveyor/test"
+	ptu "github.com/prometheus/client_golang/prometheus/testutil"
+)
+
+func TestServiceObservation_Load(t *testing.T) {
+	sc := st.NewSuperCluster(t)
+	defer sc.Shutdown()
+
+	opt := getTestOptions()
+	obs, err := NewServiceObservation("testdata/observations/good.json", *opt)
+	if err != nil {
+		t.Fatalf("observation load error: %s", err)
+	}
+	obs.Stop()
+
+	_, err = NewServiceObservation("testdata/bad.json", *opt)
```

Comment:
May want to add a test with creds that won't work - e.g. creds to an account that doesn't exist.  Here's one:

```
-----BEGIN NATS USER JWT-----
eyJ0eXAiOiJqd3QiLCJhbGciOiJlZDI1NTE5In0.eyJqdGkiOiJVRDdWSlI3SDU3VURQNU9MWUtMTlFXQUg2NVVJS1lHNjdPUkdRQ1pKSEQ0R01YTzNNTUNBIiwiaWF0IjoxNTY0NTkwMTM1LCJpc3MiOiJBREtMRkQ2NkVKV1BDUjZJSk9NWVVDQk5OREs3V05BNFZEWUZCWUdFVkdEU1VVRlNXVTRGS1NWRSIsIm5hbWUiOiJ0ZXN0X3VzZXIiLCJzdWIiOiJVREdEVUY2TExQNjdDU1VVWUU3S1hXVldMSEJWQTdXVjYzM0VKVUJWVjZXUlNFRFcyUEpQNTRWMyIsInR5cGUiOiJ1c2VyIiwibmF0cyI6eyJwdWIiOnt9LCJzdWIiOnt9fX0.gOzCljeQGcuMMxipwJsiUlcX9YfkHQyr2Bwe_kktnqBHsxvJNXv75XMLZJofXerqaVPnvdCvwSnq18mH9y-0CA
------END NATS USER JWT------

************************* IMPORTANT *************************
NKEY Seed printed below can be used to sign and prove identity.
NKEYs are sensitive and should be treated as secrets.

-----BEGIN USER NKEY SEED-----
SUAKOK4XRJRNF2B5Q6SUSQHUV3JXQC2WCQZY4MXEZH4ESN43VQKTKPB2GI
------END USER NKEY SEED------

*************************************************************
```

Reply:
tests added, also about missing json